### PR TITLE
Add more detail re timings

### PR DESCRIPTION
**Pull request by rwdavies**

### CODE REVIEW

**Comment by rwdavies** [comment]
- Todo (ideally before pushing): get talk titles from Graham and Mara
- Confirm timing start on Wednesday is OK (start 5 minutes later, see detailed program, and overview)

File: information/index.html [hunk 1]
```diff
@@ -119,8 +119,9 @@ <h3>Accommodation</h3>
 maps street view of entry</a>). 
     </p>    
     
-    <p>Please note that those who booked 4 nights of accommodation are checked in from 
-    March 27th through March 31st, while those who booked 3 nights of accommodation are 
+    <p>Please note that those who booked 4 nights of accomodation are checked in from
```
Comment (jeromekelleher) [inline information/index.html]: This is a regression - Ben fixed the spelling errors here

File: program/index.html [hunk 2]
```diff
@@ -475,14 +528,28 @@ <h2 style="text-align:center">Wednesday March 30</h2>
...
-	<td class="pd">1:00</td>
+	<td class="pd">8:05</td>
+	<td class="pd">1:05</td>
 	<td colspan="2">Welcome</td>
 </tr>
 
+<tr>
+    <td></td>
+    <td class="pd">1:05</td>
+    <td><i>Organizers</i></td>
```
Comment (jeromekelleher) [inline program/index.html]:
```suggestion
    <td><i>Organisers</i></td>
```

File: program/index.html [hunk 3]
```diff
@@ -197,6 +209,26 @@ <h2 style="text-align:center">Monday March 28</h2>
 	<td colspan="2">Welcome and plenary</td>
 </tr>
 
+<tr>
+    <td></td>
+    <td class="pd">1:00</td>
+    <td><i>Organizers</i></td>
```
Comment (jeromekelleher) [inline program/index.html]:
```suggestion
    <td><i>Organisers</i></td>
```

File: program/index.html [hunk 4]
```diff
@@ -340,6 +372,27 @@ <h2 style="text-align:center">Tuesday March 29</h2>
...
```
Comment (jeromekelleher) [inline program/index.html]:
```suggestion
    <td><i>Organisers</i></td>
```

File: program/index.html [hunk 5]
```diff
@@ -197,6 +209,26 @@ <h2 style="text-align:center">Monday March 28</h2>
...
+    <td>Housekeeping</td>
+</tr>
+
+<tr>
+    <td></td>
+    <td class="pd">1:05</td>
+    <td><i>John Todd</i></td>
+    <td>About the Welcome Centre for Human Genetics (WCHG) and speaker introduction</td>
```
Comment (jeromekelleher) [inline program/index.html]: Could drop the "and speaker introduction" here?

File: program/index.html [hunk 6]
```diff
@@ -197,6 +209,26 @@ <h2 style="text-align:center">Monday March 28</h2>
...
+</tr>
+
+<tr>
+    <td></td>
+    <td class="pd">1:15</td>
+    <td><i>Mara Lawniczak</i></td>
+    <td>Plenary title</td>
```
Comment (jeromekelleher) [inline program/index.html]: How about "Title TBC"? We want to merge ASAP and can update as we get the details. Don't want this diff lying around for days.

**Review by jeromekelleher — COMMENTED** [review]
LGTM

A few minor comments. Need to drop the changes to the info page, though.

**Comment by jeromekelleher** [comment]
I had to get this merged in a hurry so made some changes. Just minor tweaks

**Pull request merged**